Spring-Project/src/main/java/com/epam/project/spring/repository/impl/UserRepositoryImpl.java

```diff
@@ -0,0 +1,46 @@
+package com.epam.project.spring.repository.impl;
+
+import com.epam.project.spring.model.User;
+import com.epam.project.spring.repository.UserRepository;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class UserRepositoryImpl implements UserRepository {
+
+    private final List<User> listUser = new ArrayList<>();
+
+    @Override
+    public User getUser(String email) {
+        return listUser.stream().filter(user -> user.getEmail().equals(email)).findFirst().orElseThrow(() -> new RuntimeException("User not found"));
```

Comment:
Please introduce custom Exception **UserNotFound** extending **RuntimeException**. Thank you

Reply:
Thanks. Resolved